src/form/acp.jl

```diff
@@ -102,23 +102,23 @@ p[t_idx] == z*(g*v[t_bus]^2 + (-g*tr-b*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[t_bus]-t[
 q[t_idx] == z*(-(b+c/2)*v[t_bus]^2 - (-b*tr+g*ti)/tm*(v[t_bus]*v[f_bus]*cos(t[f_bus]-t[t_bus])) + (-g*tr-b*ti)/tm*(v[t_bus]*v[f_bus]*sin(t[t_bus]-t[f_bus])))
 ```
 """
-function constraint_ohms_tp_yt_to_on_off(pm::GenericPowerModel{T}, n::Int, h::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractACPForm
-    p_to  = var(pm, n, h,  :p, t_idx)
-    q_to  = var(pm, n, h,  :q, t_idx)
-    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.phase_ids(pm)]
-    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.phase_ids(pm)]
-    va_fr = [var(pm, n, j, :va, f_bus) for j in PMs.phase_ids(pm)]
-    va_to = [var(pm, n, j, :va, t_bus) for j in PMs.phase_ids(pm)]
-    z = var(pm, n, h, :branch_z, i)
+function constraint_ohms_tp_yt_to_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractACPForm
+    p_to  = var(pm, n, c,  :p, t_idx)
+    q_to  = var(pm, n, c,  :q, t_idx)
+    vm_fr = [var(pm, n, i, :vm, f_bus) for i in PMs.conductor_ids(pm)]
+    vm_to = [var(pm, n, i, :vm, t_bus) for i in PMs.conductor_ids(pm)]
```

Comment:
may want to change these `i` to `d` in `vm_fr` and `vm_to` as well so they don't collide with the `i` in the function arguments.